conf/broker.conf

```diff
@@ -1230,6 +1230,9 @@ webSocketMaxTextFrameSize=1048576
 
 ### --- Metrics --- ###
 
+# Whether the '/metrics' endpoint requires authentication. Defaults to false
+authenticateMetricsEndpoint=false
```

Comment:
Leaving this `false` is backwards compatible, but I wonder if we should make the "breaking" change and enable authentication by default here? Note that this config only takes effect when authentication is enabled.

Reply:
We can discuss this parameter later, I think it is reasonable to open.